openedx/features/portfolio_project/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""
+Portfolio project helpers and settings
+"""
+from openedx.core.djangoapps.waffle_utils import (
+    CourseWaffleFlag, WaffleFlag, WaffleFlagNamespace
+)
+
+# Namespace for learner analytics waffle flags.
+WAFFLE_FLAG_NAMESPACE = WaffleFlagNamespace(name='portfolio_project')
+
+
+# In case modal breaks Course Home or Track Selection
+# https://openedx.atlassian.net/browse/LEARNER-3926
+INCLUDE_PORTFOLIO_PROJECT_UPSELL_MODAL = WaffleFlag(WAFFLE_FLAG_NAMESPACE, 'include_upsell_modal')
+
+# Simple safety valve in case the modal breaks dashboard page.
+ENABLE_DASHBOARD_PORTFOLIO_PROJECT_UPSELL_MODAL = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'enable_dashboard_upsell_modal')
```

Comment:
Can you explain why this needs a separate flag?  I think we just want `INCLUDE_PORTFOLIO_UPSELL_MODAL` to wrap all the Portfolio modals for inclusion on the page.